test/metabase/api/persist_test.clj

```diff
@@ -43,3 +45,39 @@
         (is (= default-cron
                (get-in (task.persist-refresh/job-info-by-db-id)
                        [(:id db) :schedule])))))))
+
+(deftest persisted-info-by-id-test
+  (with-setup db
+    (t2.with-temp/with-temp
+      [:model/Card          model {:database_id (u/the-id db), :dataset true}
+       :model/PersistedInfo pinfo {:database_id (u/the-id db), :card_id (u/the-id model)}]
+      (testing "Should require a non-negative persisted-info-id"
+        (is (= "API endpoint does not exist."
+               (mt/user-http-request :crowberto :get 404 (format "persist/%d" -1)))))
+      (testing "Should not get info when the persisted-info-id doesn't exist"
+        (is (= "Not found."
+               (mt/user-http-request :crowberto :get 404 (format "persist/%d" Integer/MAX_VALUE)))))
+      (testing "Should get info when the ID exists"
+        (is (=? {:active true
+                  :card_id (u/the-id model)
+                  :id (u/the-id pinfo)
+                  :state "persisted"}
+                 (mt/user-http-request :crowberto :get 200 (format "persist/%d" (u/the-id pinfo)))))))))
+
+(deftest persisted-info-by-card-id-test
+  (with-setup db
+    (t2.with-temp/with-temp
+      [:model/Card          model {:database_id (u/the-id db), :dataset true}
+       :model/PersistedInfo pinfo {:database_id (u/the-id db), :card_id (u/the-id model)}]
+      (testing "Should require a non-negative card-id"
+        (is (= "API endpoint does not exist."
+               (mt/user-http-request :crowberto :get 404 (format "persist/card/%d" -1)))))
+      (testing "Should not get info when the card-id doesn't exist"
+        (is (= "Not found."
+               (mt/user-http-request :crowberto :get 404 (format "persist/card/%d" Integer/MAX_VALUE)))))
+      (testing "Should get info when the ID exists"
+        (is (=? {:active true
+                  :card_id (u/the-id model)
+                  :id (u/the-id pinfo)
+                  :state "persisted"}
```

Comment:
```suggestion
        (is (=? {:active  true
                 :card_id (u/the-id model)
                 :id      (u/the-id pinfo)
                 :state   "persisted"}
```

Reply:
Ah, I should have caught that.

Committed in https://github.com/metabase/metabase/pull/33776/commits/d91818564946bacef1ea7fee0bd06950166e3c5d.